tests/extract_outline_test.py

```diff
@@ -0,0 +1,198 @@
+import os
+from pdfminer.pdfdocument import PDFDocument, PDFNoOutlines
+from pdfminer.pdfpage import PDFPage
+from pdfminer.pdfparser import PDFParser
+from tools import pdf2txt
+from tools import dumppdf
+
+introduction = """Introduction 
```

Comment:
Constants should be written in capital case (see [PEP-8 spec](https://peps.python.org/pep-0008/#constants)). Also, it would be useful to have a short description about what the constant is used for.